Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -119,7 +122,26 @@ struct SettingsView: View {
                     }
                 }
                 // swiftlint:disable:next line_length
-                Text("Version \(Bundle.main.versionNumber) (\(Bundle.main.buildNumber))", comment: "Label to display version and build number.")
+                Text(LocalizedStringKey(stringLiteral: appVersion), comment: "Label to display version and build number.")
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .onLongPressGesture {
+                        let clipboard = UIPasteboard.general
+                        clipboard.setValue(appVersion, forPasteboardType: UTType.plainText.identifier)
+                        copiedAppVersion = true
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+                            copiedAppVersion = false
+                        }
+                    }
+                    .overlay {
+                        // A toast view to notify the user of version copy
+                        Text("Copied!", comment: "Text to notify user that app version was copied").font(.callout)
+                            .padding(8)
+                            .background(.black)
+                            .foregroundStyle(.white)
```

Comment:
Can you change these so that they automtaically change colors depending on the color scheme?

It was called semantic colors in iOS 13, I'm not sure if there's a `SwiftUI.Color` equivalent

Otherwise it looks like this:

![Simulator Screenshot - iPhone 15 Pro - 2023-10-11 at 13 05 42](https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/22946902/16585dfa-d413-4c83-b7f5-105ab7cbb433)